agents/python/configurator/version.py

```diff
@@ -0,0 +1,2 @@
+# the content of the file is replaced with the real version in the odiglet DOCKERFILE
+VERSION = "development"
```

Comment:
looks like the convention in python is to call it `__version__`

example:
https://github.com/open-telemetry/opentelemetry-python/blob/71416e9f40bbbf07749e975ad40f959ca750352d/opentelemetry-api/src/opentelemetry/version.py#L15